LINK/etc/profile.d/evm.sh

```diff
@@ -13,4 +13,6 @@ function tailmiq() #If no value is given with tailmiq it defaults to the evm.log
 alias tailpglog='tail -f $APPLIANCE_PG_DATA/pg_log/postgresql.log'
 
 # Appliance Status:
-alias apstatus='echo "EVM Status:";service evmserverd status;echo " ";echo "HTTP Status:";service httpd status'
+alias apstatus='echo;systemctl status evmserverd -n 0;echo; \
+systemctl status httpd.service -n 0;echo;systemctl status memcached -n 0; \
+echo -e "\nPostgreSQL service:"; /opt/rh/$APPLIANCE_PG_SCL_NAME/root/usr/bin/pg_isready'
```

Comment:
Good point, updated!   Also I remember that I did not want to use systemd for postgres as it appears to be version specific "systemctl status rh-postgresql94-postgresql".